guide/english/blockchain/index.md

```diff
@@ -30,7 +30,7 @@ As of 2016, blockchain 2.0 implementations continue to require an off-chain orac
 
 ### Bitcoin
 
-Bitcoin’s main problem to solve was to achieve distributed consensus while remaining a decentralized system. Distributed consensus is essentially the idea that all the nodes that make up the network must come to a collective agreement without the use of a centralized control. In Bitcoin, the nodes are agreeing on exactly which transactions present on the network are valid. This is difficult in many ways, one of which is the potential for n number of nodes to be malicious and not want to follow
+The main problem which Bitcoin confronted was how to achieve distributed consensus while remaining a decentralized system. Distributed consensus is essentially the idea that all the nodes that make up the network must reach a collective agreement without the use of a centralized control. In Bitcoin, the nodes agree on exactly which transactions present on the network are valid. This is difficult in many ways, one of which is the potential for n number of nodes to be malicious and not want to follow
```

Comment:
```suggestion
The main problem Bitcoin faced was how to achieve a distributed consensus while remaining a decentralized system. Distributed consensus is essentially the idea that all the nodes that make up the network must reach a collective agreement without the use of a centralized control. In the Bitcoin blockchain, the nodes must agree on which transactions present on the network are valid. This is difficult in many ways, one of which is the potential for `n` number of nodes to be malicious and not want to follow
```

The suggestions above are mainly subjective ones, and so if you disagree with me on them let me know or just add the changes you like.